docs/ispc.rst

```diff
@@ -2693,6 +2693,36 @@ the section `Logical and Selection Operations`_, which introduces helper
 functions in the standard library that provide these operations without
 short-circuiting.
 
+ISPC also enables optimizations for signed integer expressions, like in C/C++,
```

Comment:
Please remove the changes in `docs/ispc.rst`, I'll contribute them as a separate patch - either in this or other PR.

Reply:
Done.